src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -76,6 +78,9 @@ protected OptionParser buildOptionParser() {
 
 		parser.accepts( "producer", "producer type: stdout|file|kafka" ).withRequiredArg();
 		parser.accepts( "output_file", "output file for 'file' producer" ).withRequiredArg();
+		parser.accepts( "output_includes_binlog_position", "produced records include binlog position" );
+		parser.accepts( "output_includes_commit", "produced records include commit" );
+		parser.accepts( "output_includes_xid", "produced records include xid" );
```

Comment:
leave out the word `includes` -- so `output_binlog_position`, `output_commit_info`